dapp/src/utils/math.js

```diff
@@ -96,3 +96,32 @@ export async function displayCurrency(balance, contract) {
   if (!balance) return
   return ethers.utils.formatUnits(balance, await contract.decimals())
 }
+
+export function checkValidInputForCoin(amount, coin) {
+  if (amount === '') {
+    amount = '0.00'
+  }
+
+  const COIN = coin.toLowerCase()
```

Comment:
Sorry for being late to the review party.
This constant is not used anywhere? It should be used in the switch statement below.

Also I think our coding style tens to use all capital for hardcoded constants and camel case for everything else. So perhaps rename COIN to coinSymbol or equivalent.